cli/Cargo.toml

```diff
@@ -78,3 +81,10 @@ test_util = { path = "../test_util" }
 
 [target.'cfg(unix)'.dev-dependencies]
 pty = "0.2.2"
+
+[package.metadata.winres]
+# On Windows, this section defines the metadata that appears in the Deno.exe Portable Executable header
+OriginalFilename = "Deno.exe"
+LegalCopyright = "Copyright © 2018-2020 the Deno authors. All rights reserved. MIT license."
```

Comment:
Not sure what's correct here so this is just a side note.

Was poking around on my windows VM and from Microsoft products (notepad, explorer) etc this doesn't contain a year, it's just "Copyright Microsoft, All Rights Reserved."

Reply:
Depends on what the deno project prefers really.

PuTTY has: Copyright © 1997 - 2017 Simon Tatham
RealVNC has: Copyright © 2002-2020 RealVNC Ltd.
nodejs has: Copyright Node.js contributors. MIT license.
Wireshark has: Copyright © 2000 Gerald Combs <gerald@wireshark.org>
Firefox has: ©Firefox and Mozilla Developers: available under the MPL 2 license.
Chrome has: Copyright 2019 Google LLC. All rights reserved.

I guess I'll remove the "Copyright" and the years, but keep the ©